DeeployTest/test_platforms.py

```
@@ -358,6 +374,64 @@ def test_siracusa_tiled_kernels_l2_doublebuffer(test_params, deeploy_test_dir, t
+@pytest.mark.siracusa_tiled
+@pytest.mark.kernels
+@pytest.mark.singlebuffer
+@pytest.mark.l3
+@pytest.mark.parametrize(
+    "test_params",
+    generate_test_params(L3_SINGLEBUFFER_KERNELS, "L3-singlebuffer"),
+    ids = param_id,
+)
+def test_siracusa_tiled_kernels_l3_singlebuffer(test_params, deeploy_test_dir, toolchain, toolchain_dir, cmake_args,
+                                                skipgen, skipsim) -> None:
+    test_name, l1, config_name = test_params
```

Ruff (0.15.2)

[warning] Unpacked variable `config_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)